disperser/dataapi/v2/server_v2_test.go

```diff
@@ -1397,9 +1394,9 @@ func TestCheckOperatorsReachability(t *testing.T) {
 	operatorId := "0xa96bfb4a7ca981ad365220f336dc5a3de0816ebd5130b79bbc85aca94bc9b6ab"
 	mockSubgraphApi.On("QueryOperatorInfoByOperatorIdAtBlockNumber").Return(operatorInfoV2, nil)
 
-	r.GET("/v2/operators/reachability", testDataApiServerV2.CheckOperatorsReachability)
+	r.GET("/v2/operators/liveness", testDataApiServerV2.CheckOperatorsLiveness)
```

Comment:
Need to update node v2 reachability checkPath
https://github.com/Layr-Labs/eigenda/blob/master/node/node.go#L52

Reply:
Done